packages/nimble-components/src/wafer-map/modules/wafer-map-validator.ts

```diff
@@ -40,4 +43,48 @@ export class WaferMapValidator {
         }
         return !this.invalidGridDimensions;
     }
+
+    public validateDiesTableSchema(): boolean {
+        this.invalidDiesTableSchema = false;
+        if (this.wafermap.diesTable === undefined) {
+            this.invalidDiesTableSchema = false;
+        } else if (
+            this.wafermap.diesTable.numCols < 3
+            || this.wafermap.diesTable.schema.fields.findIndex(
+                f => f.name === 'colIndex'
+            ) === -1
+            || this.wafermap.diesTable.schema.fields.findIndex(
+                f => f.name === 'rowIndex'
+            ) === -1
+            || this.wafermap.diesTable.schema.fields.findIndex(
+                f => f.name === 'value'
+            ) === -1
+        ) {
+            this.invalidDiesTableSchema = true;
+        } else {
+            const colIndex = this.wafermap.diesTable.schema.fields.findIndex(
+                f => f.name === 'colIndex'
+            );
+            const rowIndex = this.wafermap.diesTable.schema.fields.findIndex(
+                f => f.name === 'rowIndex'
+            );
+            const value = this.wafermap.diesTable.schema.fields.findIndex(
+                f => f.name === 'value'
+            );
+            if (
+                !DataType.isInt(
+                    this.wafermap.diesTable.schema.fields[colIndex]!.type
+                )
+                || !DataType.isInt(
+                    this.wafermap.diesTable.schema.fields[rowIndex]!.type
+                )
+                || !DataType.isFloat(
+                    this.wafermap.diesTable.schema.fields[value]!.type
+                )
```

Comment:
I don't think these checks include the size checks as specified in the spec, i.e. Int32 and Float64. Can you make sure they are checking size as well and include tests?